.docs/user-guide/automation.md

```diff
@@ -14,7 +14,32 @@ orchestration tools below.  Optionally, Docker is also listed in some examples.
 
 <br>
 ## Ansible
-ToDo
+With Ansible, installing the latest REX-Ray binaries can be accomplished by
+including the `codenrhoden.rexray` role from Ansible Galaxy.  The role accepts
+all the necessary variables to properly fill out your `config.yml` file.
```

Comment:
Hi @codenrhoden,

The `config.yml` file to which you're referring -- is this the REX-Ray config file? If so, I'm not sure the `hosts` and `roles` keys should be at the root in your example below. Unless this is required by the driver, I'd prefer it if you refactor things so that you're not using the root namespace. Maybe shuffle them under a namespace called `ansil`?